kms/quickstart/quickstart.go

```diff
@@ -12,11 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// [START kms_quickstart]
```

Comment:
Newline after region tag so it doesn't show up in the GoDoc.

Reply:
Done